Cheza-Dzabala/way-farer
# #167705552, #167704118, #167128171 restrict admin access

#### What does this PR do?

- A user shouldn't be able to see the admin, section in the UI templates
- Implemented JS to restrict access and unauthenticated users.

#### Description of Task to be completed?

- `/admin.html` path should be blocked off unless user has is_admin: true property set.
- `/admin/* ` paths should be blocked off unless user has is_admin: true property set
- Display a 403 - Forbidden page if user is not an admin.

#### How should this be manually tested?

UI has two accounts in an array.

```javascript
const users = [
    {
        "id": 1,
        "email": "cheza@gmail.com",
        "password": "Runfree8418_!*",
        "first_name": "Cheza",
        "last_name": "Dzabala",
        "is_admin": true
    },
    {
        "id": 2,
        "email": "johndoe@gmail.com",
        "password": "Runfree8418_!*",
        "first_name": "John",
        "last_name": "Doe",
        "is_admin": false
    }
]
```

In Github pages log in to either accounts and check if admin pages are accessible. 

#### Any background context you want to provide?

All javascript is written in Vanilla javascript.

#### What are the relevant pivotal tracker stories?
#167705552, #167704118, #167128171

#### Screenshots (if appropriate)

![image](https://user-images.githubusercontent.com/13886438/62516543-ad29a880-b825-11e9-83c2-c78d54eaf86a.png)
<img width="1914" alt="Screenshot 2019-08-06 at 8 37 47 AM" src="https://user-images.githubusercontent.com/13886438/62516585-c7fc1d00-b825-11e9-9349-8bc84e76589c.png">
<img width="1915" alt="Screenshot 2019-08-06 at 8 37 57 AM" src="https://user-images.githubusercontent.com/13886438/62516587-c894b380-b825-11e9-82bb-85b6963cfba4.png">
<img width="1920" alt="Screenshot 2019-08-06 at 8 38 12 AM" src="https://user-images.githubusercontent.com/13886438/62516589-c894b380-b825-11e9-9a27-bcdf739ee1a1.png">
<img width="1913" alt="Screenshot 2019-08-06 at 8 38 35 AM" src="https://user-images.githubusercontent.com/13886438/62516590-c92d4a00-b825-11e9-9c7d-6dc48ebf57d2.png">

#### Questions:
